packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -11,19 +11,29 @@ interface ComboboxContentProps {
   readonly options: string[];
```

Comment:
unrelated, but probs should ask it now. 

@ZakaryH with your time as a person who implements Atlantis, for components and UI's like this, is it important to know the id of the thing you're selecting too? 

I know on a user selector for chips, a team needed to have the ID on the data as well since finding the ID based on the name (label) is not fun.

Reply:
it is! we've changed the shape of `options` in another PR to include an id now

cause yeah it's entirely possible for you to have two clients with the name `Jimothy Stevens`, and while that might be confusing to choose from in a list, we do need to have some way to know which one we selected

then it'd be up to the consumer to figure out which is the right one if all we're showing is the name, but that wouldn't be a concern of this component. not yet anyway!